src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -7366,6 +7417,7 @@ use the \`id\` attribute, consider setting it on a parent element instead.",
       "description": "Specifies flash messages that appear in the same order that they are listed.
 The value is an array of flash message definition objects.
 A flash message object contains the following properties:
+* \`analyticsMetadata\` (AnalyticsMetadata) - (Optional) Specifies additional analytics related metadata.
```

Comment:
(minor) i'd recommend to hyphenate: "analytics-related"

Reply:
Changed.